arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -571,6 +575,53 @@ def test_plot_pair_overlaid(models, kwargs):
     assert ax is ax2
     assert ax.shape
 
+@pytest.mark.parametrize(
+    "kwargs", [{"var_names": ["mu", "theta"]}]
+)
+def test_plot_pair_combinedims(models, kwargs):
+    ax = plot_pair(models.model_1, combine_dims=["school"], **kwargs)
+    assert np.all(ax)
+
+
+@pytest.mark.parametrize(
+    "kwargs", [{"kind": "scatter"}, {"kind": "kde"}, {"kind": "hexbin", "colorbar": True}]
+)
+def test_plot_pair_2var_combinedims(discrete_model, fig_ax, kwargs):
+    _, ax = fig_ax
+    ax = plot_pair(discrete_model, combine_dims=["school"], ax=ax, **kwargs)
+    assert ax
+
+
+def test_plot_pair_bad_combinedims(models):
+    with pytest.raises(ValueError):
+        plot_pair(models.model_1, combine_dims=["school"], kind="bad_kind")
+    with pytest.raises(Exception):
+        plot_pair(models.model_1, var_names=["mu"], combine_dims=["school"])
```

Comment:
I don't think it's necessary to check for errors adding a `combined_dims`, failing or not based on kind or var_names doesn't depend on `cobine_dims` so I think this is redundant with the original test. But I'lll deffer to our test expert @canyon289